src/InnoSetup.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="InnoSetup"
+   name="InnoSetup license" listVersionAdded="3.26">

```

Comment:
Would suggest "Inno Setup License" -- three words rather than two to reflect the actual license's title, and capital "L" also since we usually capitalize "License" as last word for licenses' full names on the list.